api/definitions.yaml

```diff
@@ -133,8 +133,8 @@ CreatedMessageWithContent:
   properties:
     id:
       type: string
-    fiscal_code:
```

Comment:
these APIs have already been published in December and we already have a few external teams developing integrations - my suggestion is to delay this change for the v2 of the APIs